tests/everest/test_data/open_shut_state_modifier/eclipse/model/INIT.SCH

```diff
@@ -0,0 +1,23 @@
+WCONPROD
+ WELL-1 SHUT BHP 5* 100 /
+ WELL-2 SHUT BHP 5* 100 /
+ WELL-3 SHUT BHP 5* 100 /
+ WELL-4 SHUT BHP 5* 100 /
```

Comment:
Just a small comment, is this file needed?

Reply:
Yes it is needed for `test_state_modifier_workflow_run`. It was a case of two wrongs were making a right so the test was not testing anything because the expected resulting file was never created due to this file being removed.  

I added it back and also added a guard against this happening in the future for this particular test.